configs/sst_virtualization_windows-subsystem.yaml

```diff
@@ -8,45 +8,40 @@ data:
   packages:
   - virtio-win
```

Comment:
Did you want to leave this empty...

```
packages: []
```

... as `virtio-win` is not in Fedora?